### PR TITLE
Patch glibc test runner

### PR DESCRIPTION
**Pull request by asvrada**
Store pid returned from fork in a local variable instead of global one. This would reduce the flakiness of glibc tests

### CODE REVIEW

File: tests/glibc/Makefile [hunk 1]
```diff
@@ -41,7 +41,9 @@ build: glibc
 PWD=$(shell pwd)
 
 $(APPDIR):
-	$(APPBUILDER) -i mystikos/glibc-test:0.1
+	# $(APPBUILDER) -i mystikos/glibc-test:0.1
+	# todo: replace with pre-built image
+	$(APPBUILDER) -i hullcritical/glibc-test:0.1
```
Comment (paulcallen) [inline tests/glibc/Makefile:46]: why did you change from `mystikos/glibc-test:0.1` to `hullcritical/glibc-test:0.1`?

Comment (asvrada) [reply tests/glibc/Makefile:46]: This image (uploaded to my personal account, will talk to @rs-- to have it replaced with a Mystikos one), is created from the modified Dockerfile to reflect the change to glibc test runner

**Review by paulcallen — APPROVED** [review]
